src/scss/includes/components/poiItem.scss

```diff
@@ -0,0 +1,37 @@
+.poiItem {
+  padding: 16px 30px;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+
+  @media (max-width: 640px) {
+    padding: 12px 16px;
+  }
+
+  .poiTitleImage {
+    width: 65px;
+    height: 65px;
+  }
+
+  &-left {
+    min-width: 0;
+
+    &-address {
```

Comment:
You have to put `.&-address` and `.&-reviews` outside of `.&-left`, otherwise the `&` will refer to `.poiItem-left`, not `.poiItem`.

Reply:
my bad yes, I fixed it